health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/UserAccessMgmtTableWrapper.js

```
@@ -0,0 +1,232 @@
+                fontWeight: "400",
+                width: "100%",
+                lineHeight: "16px",
+                color: "#C84C0E",
+              }}
+            />
+          )}
+           {showPopUp && (
+            <Wrapper
+              setShowPopUp={setShowPopUp}
+              alreadyQueuedSelectedState={row.planData.jurisdiction}
+            ></Wrapper>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)